### PR TITLE
ktLint configuration added

### PR DESCRIPTION
**Pull request by davyskiba**

### CODE REVIEW

File: app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt [hunk 1]
```diff
@@ -70,29 +71,13 @@ fun LocationUpdateBottomSheetContentPlaceholder() = AATSubscriberDemoTheme {
...
-            R.string.trackable_location_last_interval to
-                    locationUpdateBottomSheetData.lastLocationUpdateInterval?.toIntervalWithUnit(),
-            R.string.trackable_location_average_interval to
-                    locationUpdateBottomSheetData.averageLocationUpdateInterval?.toIntervalWithUnit(),
-            R.string.trackable_location_desired_interval to
-                    locationUpdateBottomSheetData.resolution?.desiredInterval?.toIntervalWithUnit(),
-            R.string.trackable_location_skipped_locations to
-                    locationUpdateBottomSheetData.locationUpdate?.skippedLocations?.size,
-        )
+        cha(locationUpdateBottomSheetData)
```
Comment (KacperKluka) [inline app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt:80]: What does `cha` stand for? For me it's not obvious what this method does :thinking:

**Review by KacperKluka — CHANGES_REQUESTED** [review]
Looks good but can be even better :grin:

File: app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt (hunk 1 above)
Comment (davyskiba) [reply app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt]: Thanks, that was development leftover

File: app/build.gradle [hunk 2]
```diff
@@ -65,6 +66,18 @@ signingConfigs {
...
 
+// https://github.com/jlleitschuh/ktlint-gradle#configuration
+ktlint {
+    verbose = true // shows rule name that failed
+    android = true
+    outputColorName = 'RED' // easier to spot failures in console output
+    disabledRules = [
+            // according to a Github issue comment this issue could be resolved with Kotlin 1.4
+            // https://github.com/pinterest/ktlint/issues/527#issuecomment-705871614
+            'import-ordering',
```
[the remark this answers is not in the document]
Comment (davyskiba) [reply app/build.gradle]: Now Android Studio lets us configure import ordering, removed this line

File: app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt (hunk 1 above)
Comment (davyskiba) [reply app/src/main/java/com/ably/tracking/demo/subscriber/ui/bottomsheet/LocationUpdateBottomSheet.kt]: https://github.com/ably/asset-tracking-android-customer-app-demo/pull/15/commits/b93239d6a2cb6c0ca339c3f8df56f2342f634203

File: app/build.gradle (hunk 2 above)
[the remark this answers is not in the document]
Comment (davyskiba) [reply app/build.gradle]: https://github.com/ably/asset-tracking-android-customer-app-demo/pull/15/commits/8bc3417b155505fafb3f91d4353565ac2f00023d

**Review by KacperKluka — APPROVED** [review]
LGTM :+1:

**Pull request merged**